cypress/integration/projectSettings/project_settings.ts

```diff
@@ -1112,8 +1114,7 @@ describe("Containers", () => {
   });
   it("should be able to add a container configuration and save it", () => {
     cy.dataCy("add-button").should("be.visible");
-    cy.dataCy("add-button").click();
-
+    cy.dataCy("add-button").trigger("mouseover").click();
```

Comment:
I wish I knew why just clicking on it is not enough.

Reply:
honestly it is a little hack... 😭 but I can't replicate the failure on local, only on CI rip